src/main.py

```diff
@@ -28,7 +28,69 @@
 IO.setup(15,IO.IN)
 
 def main():
-    return 
+        #Initilize
+
+    COLD_VALVE_SERVO = 0
+    HOT_VALVE_SERVO = 1
+    kit = ServoKit(channels=16)
+
+    # ensure water valves are off, soap line off and water directed towards fauct 
+    set_cold_valve_angle(0)
+    set_hot_valve_angle(0)
+    three_way(False)
+    two_way(False)
+
+    while True:
+        # IO.input(x) returns true if path is clear
+
+        # either IR sensor detects a hand, run hand washer
+        while not IO.input(11) or not IO.input(13):
```

Comment:
We don't need a while loop here. A `if` statement is good enough since the `while true` will constantly be running. The code in here will run once triggered and if the users hand is in it again it will still run from the outer while true loop